nimare/reports/base.py

```diff
@@ -475,16 +483,19 @@ def __init__(
                 )
             elif meta_type == "IBMA":
                 # Use "z_maps", for Fishers, and Stouffers; otherwise use "beta_maps".
-                key_maps = "z_maps" if "z_maps" in self.results.estimator.inputs_ else "beta_maps"
-                maps_arr = self.results.estimator.inputs_[key_maps]
+                key_maps = (
+                    "z_maps"
+                    if "z_maps" in self.results.estimator.inputs_["raw_data"]
+                    else "beta_maps"
```

Comment:
per our conversation, the beta maps by themselves do not make much sense to compare across studies, but that can be a different pull request.